frontend/src/language/ko.ts

```diff
@@ -725,4 +739,88 @@ export const ko: LanguageStrings = {
   generatingStatus: {
     generating: "생성 중...",
   },
+  hypnosis: {
+    hypnosisControl: "최면 제어",
+    enabled: "최면 모드 활성화",
+    disabled: "최면 모드 비활성화",
+    affectionControl: "호감도 제어",
+    intimacyControl: "친밀도 제어",
+    trustControl: "신뢰도 제어",
+    romanceControl: "연애감정 제어",
+    forceLoveUnlock: "강제 연애 모드",
+    snsFullAccess: "SNS 전체 접근",
+    secretAccountAccess: "비밀 계정 접근",
+    currentLevel: "현재 수준: {{level}}%",
+    override: "최면 우선적용",
+    naturalValue: "자연스러운 값: {{level}}%",
+    hypnosisValue: "최면 값: {{level}}%",
+    settingsWarning: "최면 제어는 캐릭터의 자연스러운 감정을 무시합니다",
+    dangerousFeature: "⚠️ 이 기능은 스토리의 자연스러운 진행을 방해할 수 있습니다"
+  },
+  sns: {
+    characterListTitle: "캐릭터 목록",
+    accessDenied: "접근 권한이 없습니다. 호감도를 높여보세요.",
+    createPost: "게시물 작성",
+    editPost: "게시물 수정",
+    likes: "좋아요",
+    comments: "댓글",
+    secretAccount: "비밀 계정",
+    mainAccount: "메인 계정",
+    secretAccountWarning: "비밀 계정의 게시물입니다",
+    selectedSticker: "선택된 스티커",
+    changeSticker: "스티커 변경",
+    selectSticker: "스티커 선택",
+    noStickers: "사용 가능한 스티커가 없습니다",
+    postPlaceholder: "무엇을 생각하고 계신가요?",
+    posts: "게시물",
+    secrets: "비밀글",
+    tags: "태그",
+    postsTab: "게시물",
+    secretsTab: "비밀글",
+    tagsTab: "태그",
+    addTag: "태그 추가",
+    tagsDescription: "이 캐릭터가 자주 사용하는 태그들입니다",
+    postsCount: "{{count}}개 게시물",
+    noPostsYet: "아직 게시물이 없습니다",
+    noSecretsYet: "아직 비밀글이 없습니다",
+    noTagsYet: "아직 태그가 없습니다",
```

Comment:
It seems to not match the language keys used in the actual UI.

```suggestion
    noPostsAvailable: "아직 게시물이 없습니다",
    noSecretsAvailable: "아직 비밀글이 없습니다",
    noTagsAvailable: "아직 태그가 없습니다",
```